swagger.yaml

```diff
@@ -3931,6 +4054,74 @@ definitions:
     required:
     - self
   QueryChartsEmbeds: {}
+  DatasetReportRun:
+    properties:
+      created_at:
+        type: string
+      completed_at:
+        type: string
+      is_latest_report_run:
+        type: string
+      is_latest_successful_report_run:
```

Comment:
Not a blocker, but we should see if we can change references to `report_run` and `dataset_report_run` to `dataset_run` in this representer.